src/App.tsx

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './components/Tabs/types/tab';
 
 export const tabs = [
```

Comment:
Please add a type for the `tabs` array, like `const tabs: Tab[] = [...]` to ensure the data structure is correct.